sdk/core/azure-core/src/http/curl/curl.cpp

```diff
@@ -1065,16 +1071,6 @@ size_t CurlSession::OnRead(uint8_t* buffer, size_t count, Context const& context
   return totalRead;
 }
 
-void CurlConnection::Shutdown()
-{
-#if defined(AZ_PLATFORM_POSIX)
-  ::shutdown(m_curlSocket, SHUT_RDWR);
-#elif defined(AZ_PLATFORM_WINDOWS)
-  ::shutdown(m_curlSocket, SD_BOTH);
-#endif
-  CurlNetworkConnection::Shutdown();
-}
-
```

Comment:
This used to be there to test how we react after a connection lost.
There was a test like: https://github.com/Azure/azure-sdk-for-cpp/pull/1721/files#diff-34f6f1c9be490d635445c2e8956dc6aa5f3c6c96f96a33ba9b658fa0f766465cR183

Are we replacing that test to actually make a call to `AzureSdkHttpbinServer::Status(101)` which is expected to shutdown a connection?

All of this logic around `shutdown`  is trying to solve this issue: https://github.com/Azure/azure-sdk-for-cpp/issues/1716

I think, at some point, we started to use the `shutdown` method as a way to avoid a connection to be re-used, which is not good.  In that case, removing the `shutdown` method makes total sense, as that was something for testing only, and not expected to be used by the client implementation.

Reply:
FWIW, I keep that test working (although I'm suspicious of the test because it may cause "put" and "patch" commands to malfunction if they receive a connection: close header.

Reply:
@vhvb1989, I am going to resolve this comment thread now so that I can merge this PR.
If you'd like more follow up your question, please create a work item so that we can follow up on it later.